R/getHyperPars.R

```diff
@@ -1,19 +1,19 @@
 #' @title Get current parameter settings for a learner.
 #'
 #' @description
-#' Retrieves the current hyperparameter settings of a learner.
+#' Retrieves the current hyperparameter settings of a learner or CPO.
 #'
-#' @template arg_learner
+#' @template arg_learner_or_cpo
 #' @param for.fun [\code{character(1)}]\cr
 #'   Restrict the returned settings to hyperparameters corresponding to \code{when}
 #'   the are used (see \code{\link[ParamHelpers]{LearnerParam}}).
 #'   Must be a subset of: \dQuote{train}, \dQuote{predict} or \dQuote{both}.
 #'   Default is \code{c("train", "predict", "both")}.
 #' @return [\code{list}]. A named list of values.
 #' @family learner
+#' @family CPO
 #' @export
 getHyperPars = function(learner,  for.fun = c("train", "predict", "both")) {
```

Comment:
My bad. I got confused because it was kind of wrong before with `assertClass(learner, classes = "Learner")` and you corrected it.